.gitattributes

```diff
@@ -1,3 +1,4 @@
 *.binary filter=lfs diff=lfs merge=lfs -crlf
 data/lm/trie filter=lfs diff=lfs merge=lfs -crlf
 data/lm/vocab.txt filter=lfs diff=lfs merge=lfs -text
+data/lm/trie.ctcdecode filter=lfs diff=lfs merge=lfs -text
```

Comment:
This will change to be `data/lm/trie`, I suspect ?

Reply:
Yes, when we completely switch over.